citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/AuthorityItemsController.java

```diff
@@ -0,0 +1,56 @@
+package edu.asu.diging.citesphere.web.user.authorities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.citesphere.core.service.IAuthorityService;
+import edu.asu.diging.citesphere.core.service.ICitationManager;
+import edu.asu.diging.citesphere.model.authority.IAuthorityEntry;
+import edu.asu.diging.citesphere.model.bib.ICitation;
+import edu.asu.diging.citesphere.model.transfer.impl.Citations;
+import edu.asu.diging.citesphere.user.IUser;
+
+@Controller
+public class AuthorityItemsController {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private List<IAuthorityEntry> authorityEntries = new ArrayList();
```

Comment:
This class is a singleton, so using a global variable like this is a big no no (all request will have the same variable/share the same data).